rosdep/base.yaml

```
@@ -1411,6 +1411,13 @@ gnuplot:
 gnuplot-x11:
   debian: [gnuplot-x11]
   ubuntu: [gnuplot-x11]
+gnutls:
```

Make the key match the ubuntu package name as is our convention. 

```suggestion
libgnutls28-dev:
```

This will also require re-alphabetizing.